### PR TITLE
Register Subdomain: nooz

### PR DESCRIPTION
**Pull request by NoozAbooz**
Added the domain: nooz

### CODE REVIEW

File: domains/nooz.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Personal portfolio for nooz",
+    "repo": "https://github.com/NoozSBC/portfolio/",
+    "owner": {
+        "username": "NoozSBC",
+        "email": "zhengm58@gmail.com",
+        "discord": "Nooz#0816"
+    },
+    "record": {
+        "CNAME": "noozsbc.github.io/portfolio/"
```
Comment (VaibhavSys) [inline domains/nooz.json:10]: Invalid CNAME. CNAME cannot contain slashes.
Check the [docs](https://www.is-a.dev/docs/github_pages/) for more information regarding setting up a is-a.dev subdomain with GitHub pages.

**Review by VaibhavSys — CHANGES_REQUESTED** [review]

File: domains/nooz.json (hunk 1 above)
Comment (NoozSBC) [reply domains/nooz.json]: Fixed

**Comment by NoozSBC** [comment]
@MaskDuck

**Review by MaskDuck — CHANGES_REQUESTED** [review]

**Review by wdhdev — CHANGES_REQUESTED** [review]
404

**Pull request closed without merging**